src/main/java/seedu/address/logic/parser/AssignCommandParser.java

```diff
@@ -27,24 +29,30 @@ public class AssignCommandParser implements Parser<AssignCommand> {
      */
     public AssignCommand parse(String args) throws ParseException {
         requireNonNull(args);
-        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_IC_NUMBER, PREFIX_DEPARTMENT);
-
-        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_IC_NUMBER, PREFIX_DEPARTMENT);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, RELEVANT_PREFIXES);
+        if (!checkIcNumberPrefixPresent(argMultimap) || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, RecordCommand.MESSAGE_USAGE));
```

Comment:
Should this be `AssignCommand.MESSAGE_USAGE` instead of `RecordCommand.MESSAGE_USAGE`?

Reply:
you are right, will update it accordingly.